crates/wasi-common/src/old/snapshot_0/hostcalls_impl/fs.rs

```diff
@@ -9,8 +9,10 @@ use crate::old::snapshot_0::sys::hostcalls_impl::fs_helpers::path_open_rights;
 use crate::old::snapshot_0::sys::{host_impl, hostcalls_impl};
 use crate::old::snapshot_0::{helpers, host, wasi, wasi32, Error, Result};
 use crate::sandboxed_tty_writer::SandboxedTTYWriter;
+use crate::sys::hoststring_from_osstring;
```

Comment:
Hmm, this seems to break the protocol we currently have for dealing with different snapshots. In principle, this should be fine, however, I was expecting the `HostString` alias also re-implemented in `crate::old::snapshot_0::sys` and used from there.